types/postcss-prefix-selector/index.d.ts

```diff
@@ -1,16 +1,16 @@
-// Type definitions for postcss-prefix-selector 1.14
+// Type definitions for postcss-prefix-selector 1.15
 // Project: https://github.com/RadValentin/postcss-prefix-selector
 // Definitions by: robertmaier <https://github.com/robertmaier>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/postcss-prefix-selector
 
 interface Options {
-    prefix?: string;
-    exclude?: ReadonlyArray<string | RegExp>;
-    ignoreFiles?: ReadonlyArray<string | RegExp>;
-    includeFiles?: ReadonlyArray<string | RegExp>;
-    transform?: (prefix: Readonly<string>, selector: Readonly<string>, prefixedSelector?: Readonly<string>, file?: Readonly<string>) => string;
+    prefix: string;
+    exclude?: ReadonlyArray<string | RegExp> | undefined;
+    ignoreFiles?: ReadonlyArray<string | RegExp> | undefined;
+    includeFiles?: ReadonlyArray<string | RegExp> | undefined;
+    transform?: ((prefix: Readonly<string>, selector: Readonly<string>, prefixedSelector: Readonly<string>, file: Readonly<string>) => string) | undefined;
 }
 
 declare function postcssPrefixSelector(options: Readonly<Options>): (root: any) => string | undefined;
 
-export = postcssPrefixSelector;
+export default postcssPrefixSelector;
```

Comment:
I think that export was OK:
https://github.com/RadValentin/postcss-prefix-selector/blob/master/index.js#L1